packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -143,10 +143,15 @@ export function convertSummaryToSnapshotTreeForCreateNew(summary: ISummaryTree):
         const summaryObject = summary.tree[key];
 
         let value: OdspSummaryTreeValue;
+        // Tracks if an entry is unreferenced. Currently, only tree entries can be marked as unreferenced. If the
```

Comment:
Is it possible to move definition of `entry` up to here and the code below (in switch statement) be like this:
this.value = ...
this.unreferenced = ..;

Reply:
I couldn't find a clean way to do this since we don't have the value at this point which is a required property.